sagemaker/sagemaker.test.w

```diff
@@ -0,0 +1,12 @@
+bring expect;
+bring "./sagemaker.w" as sageMaker;
+bring util;
+
+let sm = new sageMaker.Endpoint(util.env("ENDPOINT_NAME"), util.env("INFERENCE_NAME"));
```

Comment:
How would this test work without the environment variables set?

Ideally a test should be self contained.